.php_cs.dist

```diff
@@ -10,12 +10,10 @@ $config = PhpCsFixer\Config::create()
     ->setRules([
         '@PSR2' => true,
         '@Symfony' => true,
-        'psr0' => false,
```

Comment:
rule was not enabled, no need to extra disable it

Reply:
I like to have also rules inside that we disable to be aware of it and in case a default changes we it does not change here.

Reply:
do you really expect anyone to enable PSR-0 by default nowadays ?

Reply:
Just had to look up the current state of PSR-0 and as it's deprecated, agree that we can remove it.